testsuite/integration/smoke/src/test/java/org/jboss/as/test/smoke/webservices/WSTestCase.java

```diff
@@ -98,16 +107,17 @@ public void testManagementDescription() throws Exception {
         operation.get(ModelDescriptionConstants.RECURSIVE).set(true);
 
         final ModelNode result = managementClient.getControllerClient().execute(operation);
-        Assert.assertEquals(ModelDescriptionConstants.SUCCESS, result.get(ModelDescriptionConstants.OUTCOME).asString());
-
+        assertThat(result.get(ModelDescriptionConstants.OUTCOME).asString(), is(ModelDescriptionConstants.SUCCESS));
```

Comment:
@ehsavoie all these replacements of Assert.assertEquals() with assertThat() seem pointless, and also make it hard to see exactly what has been changed. If we really 'must' do this replacement, please do so as a separate commit.